simple-mpc-current-playlist.el

```diff
@@ -30,6 +30,9 @@
 (require 'simple-mpc-vars)
 (require 'simple-mpc-utils)
 
+(defvar simple-mpc--paylist-refresh-timer nil)
```

Comment:
Probably this should be simple-mpc--p**l**aylist-refresh-timer. The downside of auto-complete :D.